ci-operator/step-registry/hypershift/mce/kubevirt/create/2.2/hypershift-mce-kubevirt-create-2.2-chain.yaml

```diff
@@ -0,0 +1,103 @@
+chain:
+  as: hypershift-mce-kubevirt-create-2.2
+  steps:
+    - as: create-hostedcluster
+      cli: latest
+      env:
+        - name: HYPERSHIFT_NODE_COUNT
+          default: "3"
+          documentation: "The number nodes to automatically create and join to the cluster."
+      commands: |-
+        set -exuo pipefail
+
+        CLUSTER_NAME="kv-nested"
+        CLUSTER_NAMESPACE=local-cluster-${CLUSTER_NAME}
+        echo "$(date) Creating HyperShift cluster ${CLUSTER_NAME}"
+      
+        # Enable wildcard routes on the management cluster
+        oc patch ingresscontroller -n openshift-ingress-operator default --type=json -p \
+          '[{ "op": "add", "path": "/spec/routeAdmission", "value": {wildcardPolicy: "WildcardsAllowed"}}]'
+
+
+        echo "$(date) Creating HyperShift cluster ${CLUSTER_NAME}"
+        /usr/bin/hypershift create cluster kubevirt \
```

Comment:
I think we want to use the hypershift cli tool shipped by MCE. RIght now (as this is currently written), we're using the one shipped by the latest hypershift-operator for the OCP version being tested on. That's not exactly the same thing. 

To use the hypershift cli tool that the MCE operator ships, i think we can download it from the cluster itself. The url can be found in the yaml of the consoleclidownload resource. An example of how to find that is below.

```oc get ConsoleCLIDownload hypershift-cli-download -o yaml```

Reply:
```
➜   oc get csv  -A         
NAMESPACE                              NAME                         DISPLAY                              VERSION   REPLACES   PHASE
multicluster-engine                    multicluster-engine.v2.2.0   multicluster engine for Kubernetes   2.2.0                Succeeded
openshift-operator-lifecycle-manager   packageserver                Package Server                       0.19.0               Succeeded
➜  oc get ConsoleCLIDownload -A
NAME                  DISPLAY NAME                                                    AGE
helm-download-links   helm - Helm 3 CLI                                               2023-02-01T10:30:24Z
oc-cli-downloads      oc - OpenShift Command Line Interface (CLI)                     2023-02-01T10:39:03Z
odo-cli-downloads     odo - Developer-focused CLI for OpenShift (Community Support)   2023-02-01T10:39:03Z
```
I did not find the cr related to hypershift-cli, @rokej Could you help to answer to it? thanks

Reply:
You need to enable the `hypershift-preview` feature in mce to get the hypershift CLI download.

Reply:
https://github.com/stolostron/hypershift-addon-operator/blob/main/docs/provision_hosted_cluster_on_mce_local_cluster.md#enabling-the-hosted-control-plane-feature

Reply:
I ignored it, thank you for your reminder!